test/jasmine/tests/gl_plot_interact_test.js

```diff
@@ -11,22 +12,98 @@ var destroyGraphDiv = require('../assets/destroy_graph_div');
  *
  */
 
+var PLOT_DELAY = 100;
+var MOUSE_DELAY = 20;
 
-describe('Test plot structure', function() {
+
+describe('Test gl plot interactions', function() {
     'use strict';
 
     afterEach(destroyGraphDiv);
 
     describe('gl3d plots', function() {
         var mock = require('@mocks/gl3d_marker-arrays.json');
+        var gd;
+
+        function mouseEventScatter3d(type, opts) {
+            mouseEvent(type, 605, 271, opts);
+        }
 
         beforeEach(function(done) {
-            Plotly.plot(createGraphDiv(), mock.data, mock.layout).then(done);
+            gd = createGraphDiv();
+            Plotly.plot(gd, mock.data, mock.layout).then(done);
         });
 
-        it('has one *canvas* node', function() {
-            var nodes = d3.selectAll('canvas');
-            expect(nodes[0].length).toEqual(1);
+        describe('scatter3d hover', function() {
+            var node, ptData;
+
+            beforeEach(function(done) {
+                gd.on('plotly_hover', function(eventData) {
+                    ptData = eventData.points[0];
+                });
+
+                setTimeout(function() {
+                    mouseEventScatter3d('mouseover');
+                    setTimeout(done, MOUSE_DELAY);
+                }, PLOT_DELAY);
+            });
+
+            it('should have', function() {
+                node = d3.selectAll('canvas');
+                expect(node[0].length).toEqual(1, 'one canvas node');
+
+                node = d3.selectAll('g.hovertext');
+                expect(node.size()).toEqual(1, 'one hover text group');
+
```

Comment:
I used the least number of `it` as possible here to reduce the number of `create` + `destroy` graph div calls.

Unfortunately, there is no way to specify a single async `setup` step for an entire in jasmine (as far as I know). 

Note that the second argument to all `to__` methods is logged on failures, which somewhat replaces the `it` titles.